pkg/platform/runtime/target.go

```diff
@@ -70,22 +71,27 @@ func (p *ProjectTarget) Trigger() string {
 	return p.trigger.String()
 }
 
+func (p *ProjectTarget) Headless() string {
```

Comment:
I don't think the runtime target should have an opinion on how this gets used. So this should just be a bool. We can then convert it to a string at the time that we know what we're using the value for.

Reply:
The only issue with this is the use case here: https://github.com/ActiveState/cli/pull/1620/files#diff-a1ef26e8e5f00016c2a2c76985086840e00bf6a87901d4f77b3d89f9cf3d05d9R89 where we are passing in the empty string as the headless value as we do not actually know if the project is headless or not. We could default to false here as it's very unlikely that the project is headless.

Reply:
Yeah let's just default if we can't know. And by definition if we don't know then it would be headless. ie. not knowing = there's no "head".